content/events/2020/05/2020-05-13-average-american-reader-needs-you-write.md

```diff
@@ -39,7 +39,7 @@ youtube_id:
 <br /><br />
 
 {{< note >}}
-[This presentation (PDF, 3.6 MB, 51 pages)](https://digital.gov/pdf/TheAverageAmericanClean_rev.pdf), crafted by Dr. Meredith Larson of the U.S. Department of Education, offers detailed information on literacy skills of U.S. adults, including digital literacy, and the implications for communicating with the public. 
+[This presentation (PDF, 3.6 MB, 51 pages)](https://s3.amazonaws.com/digitalgov/static/TheAverageAmericanClean_rev.pdf), crafted by Dr. Meredith Larson of the U.S. Department of Education, offers detailed information on literacy skills of U.S. adults, including digital literacy, and the implications for communicating with the public. 
```

Comment:
Curious, does the shortcode not work here?

Reply:
There was some line breaking occuring after the shortcode when used within the `{{< note >}}` shortcode. Providing the full path in the link renders without the line break. See note below for another example.

Reply:
Okay, good to know. Thanks!